README.md

```diff
@@ -101,6 +101,56 @@ kind of resource existing or not existing at the namespace level (i.e. Upbound
 Cloud) to read and write namespace annotations to determine which resources are
 allowed (by RBAC) in a particular namespace.
 
+## Frequently Asked Questions
+
+**Why 'composite resource'?**
+
+Composite resources typically model infrastructure - for example an SQL instance
+or a Kubernetes cluster may be modelled as a composite resource. While these are
+conceptually one resource, they're often actually composed of several underlying
+more granular resources at the cloud provider level. A "Kubernetes cluster", for
+example might actually consist of a control plane resource, a few node pool
+resources, and a few Helm chart resources. Thus the "Kubernetes cluster" is a
+composite of a control plane, node pools, and Helm charts.
+
+**What is a 'composite resource requirement'? Why does it exist?**
```

Comment:
```suggestion
**What is a 'composite resource claim'? Why does it exist?**
```

@muvaf made a pretty strong case in our most recent 1:1 that "claim" is a more appropriate term here. I know it would bring us full circle, but it's roughly synonymous with 'requirement' and does have meaning to some folks who are familiar with the concept in upstream Kubernetes.

Personally I'm not too concerned with what we call this thing, but I do think we need a name for it. I've considered "composite resource requirement", "composite resource claim", "composite resource proxy", and "public composite resource". I'm most partial to the first two, as I suspect the latter two could be easily confused with the concept of a network proxy, or as being public to the greater internet. I wrote a pretty lengthy FAQ entry explaining why it is and why it exists that might help inform anyone who wants to take a shot at a more intuitive name.